pkg/controller/envoy/ads_event.go

```diff
@@ -197,20 +197,20 @@ func (svc *ServiceEvent) handleEdsResponse(rsp *service_discovery_v3.DiscoveryRe
 		svc.ack.ResourceNames = append(svc.ack.ResourceNames, loadAssignment.GetClusterName())
 	}
 
-	svc.rqt = newAdsRequest(resource_v3.ListenerType, nil)
+	svc.rqt = newAdsRequest(resource_v3.ListenerType, nil, resp.Nonce)
```

Comment:
Same as above.

Another point from me: we donot need to send lds request if it has been sent once